### PR TITLE
Integration with devise-token-auth finished!

### PR DESCRIPTION
**Pull request by blassoto**
### Summary

- Add devise-token-auth in route login

### Trello Card

https://trello.com/c/dHG91OSM

### CODE REVIEW

File: db/migrate/20200326145838_devise_token_auth_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+      ## Lockable
+      # t.integer  :failed_attempts, :default => 0, :null => false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      ## User Info
+      t.string :name
+      t.string :nickname
+      t.string :image
```
Comment (joredudiaz) [inline db/migrate/20200326145838_devise_token_auth_create_users.rb]: you can replace this fields with the new ones and avoid create new migrations

File: db/migrate/20200326154315_add_trackable_fields_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class AddTrackableFieldsToUser < ActiveRecord::Migration[5.1]
+  def change
+    change_table :users do |t|
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.inet     :current_sign_in_ip
+      t.inet     :last_sign_in_ip
```
Comment (joredudiaz) [inline db/migrate/20200326154315_add_trackable_fields_to_user.rb]: you can put this fields on the users migration

File: db/migrate/20200326145838_devise_token_auth_create_users.rb (hunk 1 above)
Comment (blassoto) [reply db/migrate/20200326145838_devise_token_auth_create_users.rb]: Done!

File: db/migrate/20200326154315_add_trackable_fields_to_user.rb (hunk 2 above)
Comment (blassoto) [reply db/migrate/20200326154315_add_trackable_fields_to_user.rb]: Done!

**Comment by blassoto** [comment]
@aleochoam 
imagen
imagen

**Pull request merged**